CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/base/BaseNavigator.kt

```diff
@@ -42,36 +39,13 @@ abstract class BaseNavigatorImpl(
         findNavController()?.navigateUp()
     }
 
-    protected fun popBackTo(@IdRes destinationId: Int, inclusive: Boolean = false) {
-        findNavController()?.popBackStack(destinationId, inclusive)
-    }
-
```

Comment:
Should we keep this function? 🙏  I got a feeling this most likely will be used in the future, what do you think?

Reply:
In TW there're several usages for this function, so IMO I think we should keep it 🙏🏼